include/bluetooth/mesh/gen_plvl_srv.h

```diff
@@ -152,6 +152,10 @@ struct bt_mesh_plvl_srv {
 	/** User handler functions. */
 	const struct bt_mesh_plvl_srv_handlers *const handlers;
 
+#if CONFIG_BT_SETTINGS
+	/** Storage timer */
+	struct k_work_delayable store_timer;
+#endif
```

Comment:
I thought you landed on using a single timer for all models?

Reply:
I would vote for one timer, because it would save a lot of RAM when device has a set of complex models.

Reply:
I will create a seperate issue for changing this into a single timer for all models.